src/Cli/jobs/Apps.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Canvas\Cli\Jobs;
+
+use Canvas\Contracts\Queue\QueueableJobInterfase;
+use Canvas\Jobs\Job;
+use Phalcon\Di;
+use Canvas\Models\Apps as CanvasApps;
+use Phalcon\Security\Random;
+
+class Apps extends Job implements QueueableJobInterfase
+{
+    /**
+     * Realtime channel
+     *
+     * @var string
+     */
+    protected $appName;
+
+    /**
+     * Realtime event
+     *
+     * @var string
+     */
+    protected $appDescription;
+
+    /**
+     * Realtime params
+     *
+     * @var array
+     */
+    protected $params;
+
+    /**
+     * Constructor setup info for Pusher
+     *
+     * @param string $channel
+     * @param string $event
+     * @param array $params
+     */
+    public function __construct(string $appName, string $appDescription, array $params)
```

Comment:
@rwhite27 if the job is for an app we dont need to specify appName, name and descriptoin should be good